src/services/user/services/AdminUsers.js

```diff
@@ -17,15 +17,13 @@ const getRoles = () => roleModel.find()
 	.lean()
 	.exec();
 
-const getAllUsers = (ref, schoolId, role, sortObject) => ref.app.service('usersModel').find({
-	query: {
+const getAllUsers = (ref, schoolId, role, query) => ref.app.service('usersModel').find({
+	query: Object.assign({}, query, {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use an object spread instead of `Object.assign` eg: `{ ...foo }`.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5327221)